tests/lineage/test_lineage.py

```diff
@@ -122,10 +130,7 @@ def test_lineage_render(self, dag_maker):
         assert op1.inlets[0].url == f1s.format(DEFAULT_DATE)
         assert op1.outlets[0].url == f1s.format(DEFAULT_DATE)
 
-    def test_non_attr_outlet(self, dag_maker):
-        class A:
-            pass
```

Comment:
This test was important to make sure that non-attr managed classes in Inlets/outlets continue to work.

Reply:
Non attr (or actually non serializable) inlets/outlets didn't work, they were just ignored. I don't think that is correct behavioral integrity. This leads to surprises where you supplied something as an outlet somewhere and made it an inlet to something else and it never 'arrived' an you don't know why.